account_chatter/views/account_account_views.xml

```diff
@@ -17,7 +17,7 @@
             </xpath>
         </field>
     </record>
-    <record id="view_account_form" model="ir.ui.view">
+    <record id="view_account_form_inherit" model="ir.ui.view">
```

Comment:
Since this is the standard view inheritance (no groups), it could maintain the original ID, and the above could be named `view_account_form_show_chatter`

Reply:
Done!